scipy/sparse/tests/test_array_api.py

```diff
@@ -257,7 +257,7 @@ def test_spsolve(B):
 
 
 def test_spsolve_triangular():
-    X = scipy.sparse.csr_array([
+    X = scipy.sparse.csc_array([
```

Comment:
Change from CSR to CSC

Reply:
Thanks, made a parametric test with both formats in [e97868d](https://github.com/scipy/scipy/pull/17924/commits/e97868dfca28071066e0a38b3e9e41849eb4d713)